crawlers/currency_price_tw.py

```diff
@@ -8,6 +8,17 @@
 from thousandaire.crawler import BaseCrawler
 from thousandaire.data_classes import Data, Dataset
 
+def is_float(test_data):
```

Comment:
I don't really like such idea -- to check several inputs in a call.
Most of such value checks work on single input to make things clear.